modules/cid_checker/main.tf

```diff
@@ -0,0 +1,66 @@
+#### AWS s3 bucket for market deals. ####
+
+resource "aws_s3_bucket" "cid_checker_market_deals" {
+  bucket        = "${module.generator.prefix}-marketdeals-${var.bucket_name}"
```

Comment:
Nope, we can't do this that way. We have a lot of dependencies here - many teams are already depending on us having the data in a buckets with a very specific names. I have read @witnsby comments and agree that we must stick to naming convention wherever possible, however here we can't afford changing the name of the bucket because it will most likely cause tons of downtime and additional efforts for the Marketdeals consumers

Reply:
Double this, we can afford to generate it like this for calibration/wallaby, but as @ArseniiPetrovich said, it will be painful to make changes for mainnet. 
U can probably use ternary operator like - if mainnet env - just reuse old name, if other - let it me the new one generated

Reply:
Okay, we have two sentences now, as I see in bucket name = marketdeals-wallaby, marketdeals-callibration, marketdeals (mainnet).

First sentence : Provide all buckets names  "marketdeals-${var.bucket_name}"

Second sentence by @mur-me ,  do you mean this ?

bucket = var.bucket_name == local.is_mainnet_envs ? "marketdeals" : "marketdeals-${var.bucket_name}"

Reply:
Nah, let's not overcomplicate this now. Make bucket name as a module variable. Then use "marketdeals" for mainnet and  "marketdeals-calibration", "marketdeals-wallaby" for other networks.

Reply:
Was changed bucket name. Please review again.